rasterio/_io.pyx

```diff
@@ -367,16 +367,24 @@ cdef class DatasetReaderBase(DatasetBase):
                     dst_transform=self.window_transform(window),
                     resampling=resampling) as vrt:
```

Comment:
@ungarj would you mind trying `resampling=Resampling.nearest` at line 368 ? I'd like to know if you still get the results you're expecting. Anything other than nearest might be superfluous in this specific use of WarpedVRT.

Reply:
@sgillies you're right, it still makes the test pass. Should I remove it?